components/FlexibleHeader/examples/FlexibleHeaderConfiguratorExample.m

```diff
@@ -114,6 +117,23 @@ - (void)field:(FlexibleHeaderConfiguratorField)field didChangeValue:(NSNumber *)
         headerView.shiftBehavior = MDCFlexibleHeaderShiftBehaviorEnabledWithStatusBar;
         [self didChangeValueForField:FlexibleHeaderConfiguratorFieldShiftBehaviorEnabled
                             animated:YES];
+        [self didChangeValueForField:FlexibleHeaderConfiguratorFieldShiftBehaviorHideable
```

Comment:
I don't really understand this PR, but if the value changed for that field already, shouldn't some property/ivar be updated before calling the `didChange` method?

Reply:
All of the shift behavior fields are dependent on the same shiftBehavior backing ivar of `headerView.shiftBehavior`. The purpose of this code is to inform the other switches that may have been affected by a change to `headerView.shiftBehavior` that they should update their visual state accordingly.

Reply:
E.g. in this case, if the user turns on "enabled with status bar", we want "hideable" to update to show that it is now turned off.